torch/_decomp/__init__.py

```diff
@@ -262,25 +266,16 @@ def remove_decompositions(
 import torch._refs
 
 
+def core_aten_decompositions() -> "CustomDecompTable":
+    from torch.export.exported_program import default_decompositions
+    return default_decompositions()
+    
+
 # See NOTE [Core ATen Ops]
 #
 # list was copied from torch/_inductor/decomposition.py
 # excluding decompositions that results in prim ops
 # Resulting opset of decomposition is core aten ops
-def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
-    # If it is fbcode change, we return the old decomposition list
-    from torch._export.utils import (
-        _collect_all_valid_cia_ops_for_aten_namespace,
-        _get_decomp_for_cia,
-    )
-
-    # Entry without functional CIA ops
-    decomp_table = _core_aten_decompositions_post_autograd()
-    for op in _collect_all_valid_cia_ops_for_aten_namespace():
-        decomp_table[op] = _get_decomp_for_cia(op)
-    return decomp_table
-
-
 def _core_aten_decompositions_post_autograd() -> (
```

Comment:
I guess at some point later we could delete this code (and some of the other code in this file) that needed to sweat a bunch to generate the old core aten decomps?